packages/client/wallets/smart-wallet/src/error/index.ts

```diff
@@ -1,83 +1,83 @@
 import { PasskeyDisplay, SignerDisplay } from "@/types/API";
 
-export const SmartWalletErrors = {
-    NOT_AUTHORIZED: "smart-wallet:not-authorized",
-    TRANSFER: "smart-wallet:transfer.error",
-    CROSSMINT_SERVICE: "smart-wallet:crossmint-service.error",
-    ERROR_JWT_EXPIRED: "smart-wallet:not-authorized.jwt-expired",
-    ERROR_JWT_INVALID: "smart-wallet:not-authorized.jwt-invalid",
-    ERROR_JWT_DECRYPTION: "smart-wallet:not-authorized.jwt-decryption",
-    ERROR_JWT_IDENTIFIER: "smart-wallet:not-authorized.jwt-identifier",
-    ERROR_USER_WALLET_ALREADY_CREATED: "smart-wallet:user-wallet-already-created.error",
-    ERROR_OUT_OF_CREDITS: "smart-wallet:out-of-credits.error",
-    ERROR_WALLET_CONFIG: "smart-wallet:wallet-config.error",
-    ERROR_ADMIN_MISMATCH: "smart-wallet:wallet-config.admin-mismatch",
-    ERROR_PASSKEY_MISMATCH: "smart-wallet:wallet-config.passkey-mismatch",
-    ERROR_PASSKEY_PROMPT: "smart-wallet:passkey.prompt",
-    ERROR_PASSKEY_INCOMPATIBLE_AUTHENTICATOR: "smart-wallet.passkey.incompatible-authenticator",
-    ERROR_PASSKEY_REGISTRATION: "smart-wallet:passkey.registration",
-    ERROR_ADMIN_SIGNER_ALREADY_USED: "smart-wallet:wallet-config.admin-signer-already-used",
-    ERROR_PROJECT_NONCUSTODIAL_WALLETS_NOT_ENABLED: "smart-wallet:wallet-config.non-custodial-wallets-not-enabled",
-    UNCATEGORIZED: "smart-wallet:uncategorized", // catch-all error code
+export const SmartWalletErrorCode = {
+    NOT_AUTHORIZED: "not-authorized",
+    CROSSMINT_SERVICE: "crossmint-service",
+    JWT_EXPIRED: "not-authorized.jwt-expired",
+    JWT_INVALID: "not-authorized.jwt-invalid",
+    JWT_DECRYPTION: "not-authorized.jwt-decryption",
+    JWT_IDENTIFIER: "not-authorized.jwt-identifier",
+    OUT_OF_CREDITS: "out-of-credits",
+    TRANSFER: "smart-wallet:transfer",
+    SMART_WALLETS_NOT_ENABLED: "smart-wallet:not-enabled",
+    USER_WALLET_ALREADY_CREATED: "smart-wallet:user-wallet-already-created",
+    WALLET_CONFIG: "smart-wallet:config",
+    ADMIN_MISMATCH: "smart-wallet:config.admin-mismatch",
+    PASSKEY_MISMATCH: "smart-wallet:config.passkey-mismatch",
+    ADMIN_SIGNER_ALREADY_USED: "smart-wallet:config.admin-signer-already-used",
+    PASSKEY_PROMPT: "smart-wallet:passkey.prompt",
+    PASSKEY_INCOMPATIBLE_AUTHENTICATOR: "smart-wallet:passkey.incompatible-authenticator",
+    PASSKEY_REGISTRATION: "smart-wallet:passkey.registration",
+    UNCATEGORIZED: "smart-wallet:uncategorized", // smart wallet specific catch-all error code
 } as const;
```

Comment:
Key changes:
- Remove redundant ERROR prefix
- Drop `.error`
- Remove `smart-wallet` from generic errors that'll be moved to our base SDK. https://linear.app/crossmint/issue/WAL-2588/move-generic-errors-to-core-sdk